.eslintrc

```diff
@@ -79,6 +79,7 @@
         "browser": true,
         "browserAction": true,
         "createStylesheet": true,
+        "DatabaseState": true,
```

Comment:
Is "true" the right value here? Perhaps DISCONNECTED?

Reply:
That just means Eslint can find the variable and doesn't report it as error when looking at the files.